docs/users-guide/13-sql-parameters.md

```diff
@@ -137,9 +118,7 @@ Or in MongoDB:
 {% endraw %}
 ```
 
-To use multiple optional clauses you must include at least one regular `WHERE` clause followed by optional clauses, each starting with `AND`.
-
-Example:
+To use multiple optional clauses, you must include at least one regular `WHERE` clause followed by optional clauses, each starting with `AND`:
```

Comment:
Wouldn't mind if we simplified the multiple-example, so it included just one example with both simple and Field Filter.
And then also extend the example for Mongo, since it's not clear how to have multiple filters, but I would just move all Mongo stuff below the other, since it's so confusing unless you're using Mongo.